app/views/tag/_location.html.erb

```diff
@@ -15,7 +15,7 @@
       <% end %>
       if( $(this).attr("submit_to") ) {
         modalUrl += firstVar ? '?' : '&';
-        modalUrl += 'url=' + encodeURIComponent(locationInput.attr("submit_to"));
+        modalUrl += 'url=' + encodeURIComponent($(this).attr("submit_to"));
```

Comment:
Why is this change required?

Reply:
I don't know why, but this doesn't seem to work, when we select using `locationInput` it returns `undefined`. So I though if we are searching if `submit_to` exists or not using `$(this)`, we can also use the value in same way,

Reply:
Also, this change was made for the [PR](https://github.com/publiclab/plots2/pull/8988).